components/layout/Nav.js

```diff
@@ -59,10 +80,13 @@ const StyledNav = styled.nav`
     background: none;
     border: 0;
     cursor: pointer;
-    @media (max-width: 700px) {
-      font-size: 10px;
-      padding: 0 10px;
+    @media (max-width: 750px) {
+      padding: 0 1rem;
+      letter-spacing: 0.1rem;
     }
+  }
+  a.loggedOut,
+  b.loggedOut {
     &:hover,
     &:focus {
       border: 0;
```

Comment:
Why do you have this border?
and if you don't want a `border` set it to `border: none` instead of `0`, but I think this can be removed